### PR TITLE
TREV-1119 Create separate databases for tampere, vesilahti & hämeenkyrö

### PR DESCRIPTION
**Pull request by cyanidesayonara**
* Copy & extend database creation files from evaka
* Add database url to each environment's application*.yml file

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -117,10 +117,10 @@ services:
       JAVA_OPTS: -server -Djava.security.egd=file:/dev/./urandom -Xms1024m -Xss512k -Xmx1024m -XX:TieredStopAtLevel=1
       FLYWAY_USERNAME: ${EVAKA_SERVICE_FLYWAY_USERNAME}
       FLYWAY_PASSWORD: ${EVAKA_SERVICE_FLYWAY_PASSWORD}
-      FLYWAY_URL: jdbc:postgresql://${EVAKA_DATABASE_HOST}:${EVAKA_DATABASE_PORT}/${EVAKA_SERVICE_DATABASE_NAME}
+      FLYWAY_URL: jdbc:postgresql://${EVAKA_DATABASE_HOST}:${EVAKA_DATABASE_PORT}/evaka_tampere_local
```
Comment (patari) [inline docker-compose.yml:120]: Remove line, property doesn't exist.

File: docker-compose.yml [hunk 2]
```diff
@@ -117,10 +117,10 @@ services:
...
       FLYWAY_PLACEHOLDERS_MIGRATION_USER: ${EVAKA_SERVICE_FLYWAY_USERNAME}
       FLYWAY_PLACEHOLDERS_APPLICATION_USER: ${EVAKA_SERVICE_APPLICATION_USERNAME}
-      SPRING_DATASOURCE_URL: jdbc:postgresql://${EVAKA_DATABASE_HOST}:${EVAKA_DATABASE_PORT}/${EVAKA_SERVICE_DATABASE_NAME}
+      SPRING_DATASOURCE_URL: jdbc:postgresql://${EVAKA_DATABASE_HOST}:${EVAKA_DATABASE_PORT}/evaka_tampere_local
```
Comment (patari) [inline docker-compose.yml:123]: `evaka_tampere_local` -> `evaka_${EVAKA_CUSTOMIZATIONS}_local` should work for all.

File: service/src/main/resources/application-hameenkyro_evaka.yaml [hunk 3]
```diff
@@ -4,7 +4,9 @@
 
 evaka:
   database:
+    url: jdbc:postgresql://localhost:5432/evaka_hameenkyro_local
     flyway:
+      url: jdbc:postgresql://localhost:5432/evaka_hameenkyro_local
```
Comment (patari) [inline service/src/main/resources/application-hameenkyro_evaka.yaml:9]: Remove, property doesn't exist.

File: service/src/main/resources/application-tampere_evaka.yaml [hunk 4]
```diff
@@ -7,7 +7,9 @@ spring:
     name: "trevaka-service"
 evaka:
   database:
+    url: jdbc:postgresql://localhost:5432/evaka_tampere_local
     flyway:
+      url: jdbc:postgresql://localhost:5432/evaka_tampere_local
```
Comment (patari) [inline service/src/main/resources/application-tampere_evaka.yaml:12]: Remove, property doesn't exist.

File: service/src/main/resources/application-tampere_evaka.yaml [hunk 5]
```diff
@@ -20,3 +22,6 @@ evaka:
         street_address: "PL 487"
         postal_code: "33101"
         post_office: "Tampere"
+  service:
+    database:
+      name: evaka_tampere_local
```
Comment (patari) [inline service/src/main/resources/application-tampere_evaka.yaml:27]: Remove, property doesn't exist.

File: service/src/main/resources/application-vesilahti_evaka.yaml [hunk 6]
```diff
@@ -4,7 +4,9 @@
 
 evaka:
   database:
+    url: jdbc:postgresql://localhost:5432/evaka_vesilahti_local
     flyway:
+      url: jdbc:postgresql://localhost:5432/evaka_vesilahti_local
```
Comment (patari) [inline service/src/main/resources/application-vesilahti_evaka.yaml:9]: Remove, property doesn't exist.

File: service/src/test/resources/application-integration-test.yml [hunk 7]
```diff
@@ -13,7 +13,7 @@ evaka:
     fee_decisions: trevaka-paymentdecisions-it
     voucher_value_decisions: trevaka-vouchervaluedecisions-it
   database:
-    url: jdbc:postgresql://localhost:5432/evaka_it
+    url: jdbc:postgresql://localhost:5432/evaka_tampere_it
```
Comment (patari) [inline service/src/test/resources/application-integration-test.yml:16]: I think we could remove this line now, better to not have default value for url (and rely on `application-<municipality>_evaka.yaml` in tests).

**Review by patari — CHANGES_REQUESTED** [review]
Looks good. Maybe some day we should add this support to core so we don't need to maintain db settings.

File: docker-compose.yml (hunk 1 above)
Comment (cyanidesayonara) [reply docker-compose.yml]: Removed

File: service/src/main/resources/application-tampere_evaka.yaml (hunk 5 above)
Comment (cyanidesayonara) [reply service/src/main/resources/application-tampere_evaka.yaml]: Removed

File: service/src/main/resources/application-tampere_evaka.yaml (hunk 4 above)
Comment (cyanidesayonara) [reply service/src/main/resources/application-tampere_evaka.yaml]: Removed

**Review by patari — APPROVED** [review]

File: docker-compose.yml (hunk 2 above)
Comment (cyanidesayonara) [reply docker-compose.yml]: Changed and seems to be working

File: service/src/main/resources/application-vesilahti_evaka.yaml (hunk 6 above)
Comment (cyanidesayonara) [reply service/src/main/resources/application-vesilahti_evaka.yaml]: Removed

File: service/src/test/resources/application-integration-test.yml (hunk 7 above)
Comment (cyanidesayonara) [reply service/src/test/resources/application-integration-test.yml]: Also removed

**Comment by patari** [comment]
Just realized that integration tests are now against `_local` databases, not `_it` like before.

**Pull request merged**